dotnet/src/SemanticKernel.Core/Extensions/KernelSemanticFunctionExtensions.cs

```diff
@@ -47,9 +47,11 @@ public static class KernelSemanticFunctionExtensions
         var config = new PromptTemplateConfig
         {
             Description = description ?? "Generic function, unknown purpose",
-            Type = "completion",
-            Completion = requestSettings
         };
+        if (requestSettings is not null)
```

Comment:
nit: add a new line to separate the `if` block from the `config` varibale` declaration.
```suggestion

        if (requestSettings is not null)
```